gulpfile.babel.js

```diff
@@ -200,18 +200,20 @@ gulp.task('transifex-download', function (done) {
     require('./gulp/transifex-download')(paths.dest + '/locales/', done);
 });
 
-
 /**
  * Task `serve:static` - Serve dist build (for heroku)
  */
+
 gulp.task('serve:static', function() {
+    gulp.start('dist');
+
     serve.init({
         server: {
             baseDir: paths.dest
         },
         port: process.env.PORT || 3000,
         ui: false,
-        codeSync: false,
+        autoWatch: true,
```

Comment:
Why does this change?

Reply:
I literally don't know what this is and why it changed (somewhat concerning). Will change it back. I imagine I changed this to help with languages locally or something and then forgot.